src/en/juju-ha.md

```diff
@@ -51,16 +51,10 @@ state servers nor remove the failed ones. However, as long as more than half of
 the original number of state servers remain available you can manually recover.
 The process is detailed below.
 
-1. Run `juju ensure-availability`.
-1. Verify that the output of `juju status` shows a value of `has-vote` for the
```

Comment:
That works, but the attribute name has changed to "controller-member-status"

However, running 'juju enable-ha' a second time does not have the expected result. In fact, nothing changes. So, for now, it seems we can enable, which creates servers if needed, we can check to see that the "controller-member-status" is set, but we can not make Juju no longer consider machines as controller members.

Watch for an edit soon.